metabrainz/reports/annual_reports/files/2018/content.html

```diff
@@ -0,0 +1,29 @@
+<h2>Archive notice</h2>
+
+<p>
+Annual reports from 2006 to 2017 are still available for history.
```

Comment:
```suggestion
Annual reports from 2006 to 2017 are still available as historical data.
```
Maybe?

Reply:
I’m missing the difference but I’m not native English speaker. Is “history” too much here? Would “for the record” be more appropriate?
```suggestion
Annual reports from 2006 to 2017 are still available for the record.
```